### DISCUSSION TITLE
can not ping to end device from OTBR with system ping command after unplug ethernet cable

### DISCUSSION THREAD

**Discussion by xiaojiang2017** [openthread/openthread#9270] [Q&A]
**Describe the bug** A clear and concise description of what the bug is.
Plug in the ethernet cable, and the backboneIfName is set to eth0, the OTBR can ping the end device success.
```
~# ot-ctl netdata show
Prefixes:
fd03:7818:c57c:1::/64 paos low f000
Routes:
fdca:7e17:ec28:fdef::/64 s med f000
fd4e:96f4:bbd0:0::/64 s med f000
fd2e:f85e:21bc:1::/64 s med f000
fda5:d7fa:c32c:e123::/64 s med f000
fd03:7818:c57c:2:0:0::/96 sn low f000
fd89:c2d5:e159:4e59::/64 s med f000
Services:
44970 01 21000500000e10 s f000
44970 5d fd0e3e006f16667d1dbbf2af721ed9c6d11f s f000
Done
~# ping -6 -c1 fd03:7818:c57c:1:a047:bb94:ad27:a9b0
PING fd03:7818:c57c:1:a047:bb94:ad27:a9b0 (fd03:7818:c57c:1:a047:bb94:ad27:a9b0): 56 data bytes
64 bytes from fd03:7818:c57c:1:a047:bb94:ad27:a9b0: seq=0 ttl=64 time=22.608 ms

--- fd03:7818:c57c:1:a047:bb94:ad27:a9b0 ping statistics ---
1 packets transmitted, 1 packets received, 0% packet loss
round-trip min/avg/max = 22.608/22.608/22.608 ms
~# ping -6 -c1 fd0e:3e00:6f16:667d:9c7c:d613:988f:7fef
PING fd0e:3e00:6f16:667d:9c7c:d613:988f:7fef (fd0e:3e00:6f16:667d:9c7c:d613:988f:7fef): 56 data bytes
64 bytes from fd0e:3e00:6f16:667d:9c7c:d613:988f:7fef: seq=0 ttl=64 time=20.135 ms

--- fd0e:3e00:6f16:667d:9c7c:d613:988f:7fef ping statistics ---
1 packets transmitted, 1 packets received, 0% packet loss
round-trip min/avg/max = 20.135/20.135/20.135 ms
```
Unplug the ethernet cable, the OTBR can not access the end device. If I have an application running on OTBR, it won't be able to communicate with the end device. Is this expected behavior?
```
~# ot-ctl netdata show
Prefixes:
Routes:
Services:
44970 01 21000500000e10 s f000
44970 5d fd0e3e006f16667d1dbbf2af721ed9c6d11f s f000
Done
~# ping -6 -c1 fd03:7818:c57c:1:a047:bb94:ad27:a9b0
PING fd03:7818:c57c:1:a047:bb94:ad27:a9b0 (fd03:7818:c57c:1:a047:bb94:ad27:a9b0): 56 data bytes
ping: sendto: Permission denied
~# ping -6 -c1 fd0e:3e00:6f16:667d:9c7c:d613:988f:7fef
PING fd0e:3e00:6f16:667d:9c7c:d613:988f:7fef (fd0e:3e00:6f16:667d:9c7c:d613:988f:7fef): 56 data bytes

--- fd0e:3e00:6f16:667d:9c7c:d613:988f:7fef ping statistics ---
1 packets transmitted, 0 packets received, 100% packet loss
```

**To Reproduce** Information to reproduce the behavior, including:

1. Git commit id **GSDK v4.2.3**
2. IEEE 802.15.4 hardware platform
3. Build steps
4. Network topology

**Comment by jwhui** [comment]
The `fd03:7818:c57c:1::/64` prefix (aka Off-Mesh Routable or OMR prefix) is only configured when the device is serving as a Border Router. Unplugging the Ethernet cable means the device is no longer operating as a Border Router.

The OMR prefix is randomly generated by each Border Router. The OMR prefix in use depends on the set of Border Routers that are currently active. As a result, applications should be able to react to changing OMR prefixes.

When there are no Border Routers available, then all that's left is the Mesh-Local Prefix. This is similar to a Wi-Fi/Ethernet segment that falls back to just link-local addresses when the global Internet connectivity is no longer available.

**Reply by xiaojiang2017** [reply]
Hi @jwhui,
Is there an interface or command to generate a randomly valid prefix? I need to configure an OMR prefix from my application if I still want to be able to access the end device from the OTBR when the Ethernet cable is disconnected.

**Reply by xiaojiang2017** [reply]
I checked the Thread network information of the Apple Homepod mini and found that its OMR prefix is configured as the default route.
When the Homepod's wifi is disconnected, omrprefix does not unregister. So even when its wifi is disconnected, it can still control the end device. Is there any problem if every OTBR is configured as the default route?
```
uart:~$ ot dataset active
Active Timestamp: 2
Channel: 25
Channel Mask: 0x03fff800
Ext PAN ID: e589c2d5e1594e59
Mesh Local Prefix: fde5:89c2:d5e1:0::/64
Network Key: d9cca4111149b4a9aa5ed1af4d4df253
Network Name: MyHome857640893
PAN ID: 0x9710
PSKc: 0e9a03e4c99b82058f53385cb3fd33a9
Security Policy: 672 onrc
Done

uart:~$ ot netdata show
Prefixes:
fd4e:96f4:bbd0:0::/64 paros low 2400
Routes:
::/0 s med 2400
Services:
44970 01 6104b000000e10 s 2400
44970 5c9d  s 2400
44970 5d fd4e96f4bbd0000082e69fd81e712adaf082 s 2400
Done
```

**Reply by jwhui** [reply]
> Is there any problem if every OTBR is configured as the default route?

This breaks failover. If every BR is advertising a default route, but some do not actually provide connectivity to the broader IP network, there is no way for Thread devices to properly reach the broader IP network.
